File: wooseongshin/2월/22일/옹알이(1).java
class Solution {
    private static final String PATTERN = "^(aya(?!aya)|ye(?!ye)|woo(?!woo)|ma(?!ma))+$";

    public int solution(String[] babbling) {
        int answer = 0;
        for (String s : babbling) {
            if (s.matches(PATTERN)) {
                answer++;
            }
        }
        return answer;
    }
}

File: Eunleelee/2024-02/2월 22일/옹알이(1).py
def solution(babbling):
    possible = ["aya", "ye", "woo", "ma"]
    answer = 0
    child = ''
    
    for sounds in babbling:
        child = ''
        for sound in sounds:
            child += sound
            if child in possible:
                child = ''
        if len(child) == 0:
            answer += 1
    return answer
